indico/modules/events/abstracts/templates/display/tracks.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title %}
+    {% trans %}Reviewing Area{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="track-review-list">
+        {% if not tracks %}
+            <p>
+                {%- trans -%}
+                    There are currently no tracks for which you are a reviewer or convener.
+                {%- endtrans -%}
+            </p>
+        {% else %}
+            <ul>
+                {% for track in tracks %}
+                    <div class="track-review-row">
+                        <li>
+                            <div class="title">
+                                <a href="{{ url_for('abstracts.display_reviewable_track_abstracts', track) }}">
+                                    {{- track.title -}}
+                                </a>
+                                <span class="badge"
+                                      title="{{ abstract_count[track]['total'] }} {% trans %}total{% endtrans %}">
+                                    {{- abstract_count[track]['total'] - abstract_count[track]['reviewed'] }}
+                                    {% trans %}left to review{% endtrans -%}
```

Comment:
We should probably replace the "x total" with "x already reviewed" or change the logic. Right now the "total" is the number of abstracts that are being reviewed_for the track which are still in the 'submitted' state